iFixit/react-commerce
# Refactor product list url scheme

closes #163 

To implement this new url scheme I need a new `type` field in the product list (enum with values: `parts`, `tools`, `marketing`), so that when I have to generate a breadcrumb or just link to a specific product list, I know exactly what path to generate for the product list. All newly created product lists default to `parts`. Tools categories need to have `tools` type.. All other product lists should have type `marketing` (e.g. A "black-friday-sales" product list with type `marketing` would be rendered under `Store/black-friday-sales`.

cc @sterlinghirsh 

## QA

1. Visit the Vercel PR preview (replace `vercel.app` with `cominor.com`)
2. Visit parts product list page
3. Verify that the new product list spec detailed in #163 is correctly implemented
